frontend/src/app/game/pages/outro/outro.service.ts

```diff
@@ -35,7 +36,7 @@ export class OutroService {
 
   allResults$: Observable<GameResult[]>;
 
-  private fetch$ = new Subject();
+  scenarioName$ = new ReplaySubject<ScenarioName>(1);
```

Comment:
Pls, make Subject private again. Consider to use BehaviourSubject.
What about:
```
  private _scenarioName$ = new BehaviorSubject<string>('');
  scenarioName$ = this._scenarioName$.asObservable();
```
Last point: initial value?

Reply:
I converted the `Subject` to private variable as suggested though I didn't switch to `BehaviorSubject` since there is no initial value until a game is loaded so I think the current version is fine.